core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/LogErrorHandler.java

```diff
@@ -24,15 +26,34 @@ public class LogErrorHandler implements Handler<RoutingContext> {
 
   @Override
   public void handle(final RoutingContext failureContext) {
-
     if (failureContext.failed()) {
-      LOG.error(
-          String.format("Failed request: %s", failureContext.request().absoluteURI()),
-          failureContext.failure());
+      final String requestUri =
```

Comment:
Not going to block on this, but the variable name is a bit misleading as sometimes it is an error message sometimes it's the requestUri.